src/cegpy/trees/event.py

```diff
@@ -140,6 +140,9 @@ def __init__(self, dataframe, sampling_zero_paths=None,
         self.__construct_event_tree()
         logger.info('Initialisation complete!')
 
+        # creat the dot graph representation
+        self._dot_graph = None
```

Comment:
self._dot_graph appears unused. Do we need this?

Reply:
You are right, this is redundant.